pkg/client/typed/generated/extensions/unversioned/extensions_client.go

```diff
@@ -46,10 +45,6 @@ func (c *ExtensionsClient) Deployments(namespace string) DeploymentInterface {
 	return newDeployments(c, namespace)
 }
 
-func (c *ExtensionsClient) HorizontalPodAutoscalers(namespace string) HorizontalPodAutoscalerInterface {
```

Comment:
Am not sure why are these being deleted.
Extensions client should also continue to work.
Extensions client will talk to apiserver on /extensions/v1beta1/hpa.
The new autoscaling client will talk to apiserver on /autoscaling/v1/hpa.
Both can coexist.